kernel/libs/aster-bigtcp/src/socket/bound.rs

```diff
@@ -103,6 +103,18 @@ impl TcpSocket {
             self.is_dead.store(true, Ordering::Relaxed);
         }
     }
+
+    /// Sets the TCP socket as dead.
+    ///
+    /// See [`BoundTcpSocketInner::is_dead`] for the definition of dead TCP sockets.
+    ///
+    /// This method is only used when the socket is in [`TimeWait`] state.
+    ///
+    /// [`TimeWait`]: smoltcp::socket::tcp::State::TimeWait
+    fn set_dead(&self, socket: &RawTcpSocketExt) {
```

Comment:
Can we revise the method name to make it clear that it can only be used for TimeWait sockets? For example, does `set_dead_timewait` sound better?